src/intro/hello-world.md

```diff
@@ -275,8 +275,8 @@ Let's break this down.
 
 4. We define two fields `speed` and `angular_speed` for the logic. These are regular Rust fields, no magic involved. More about their use later.
 
-5. The `#[base]` attribute declares the `sprite` field, which allows `self` to access the base instance (via composition, as Rust does not have
-   native inheritance).
+5. The `#[base]` attribute declares the `sprite` field, which allows `self` to access the base instance (via `base()` or `base_mut()` as Rust does
+   not have native inheritance).
```

Comment:
The old sentence is still true -- the point is that since no inheritance is possible, we use composition. The `base()/base_mut()` are just accessors.

It still makes sense to mention those, but it should be a new sentence and not replace the previous one.